# command-line-arguments
# [command-line-arguments]
vet: /tmp/stablecode-file_exists-py-go.go:10:23: expected ';', found ')' (and 3 more errors)
